Dockerfile

```diff
@@ -0,0 +1,58 @@
+# The official python:2.7 image no longer receives automatic rebuilds (it's
+# a year old as of October, 2021), so use the latest LTS release of Ubuntu
+# that includes Python 2.7 instead.
+FROM ubuntu:20.04
+
+# Install packages needed to run your application (not build deps).
+RUN set -ex \
+    && RUN_DEPS=" \
+    ca-certificates \
+    git \
+    libpq5 \
+    make \
+    postgresql-client \
+    python2.7 \
+    " \
+    && apt-get update && apt-get install -y --no-install-recommends $RUN_DEPS \
+    && rm -rf /var/lib/apt/lists/*
+
+# Copy requirements
+ADD requirements.txt /requirements.txt
+ADD DjangoPlugin /DjangoPlugin
+
+# Install build deps, then run `pip install`, then remove unneeded build deps all in a single step.
+# Correct the path to your production requirements file, if needed.
+# For installing a python2.7-compatible pip: https://stackoverflow.com/a/54335642/166053
+# Since we are using the system Python, also isolate the code in its own virtualenv.
+RUN set -ex \
+    && BUILD_DEPS=" \
+    build-essential \
+    libpq-dev \
+    python2.7-dev \
+    wget \
+    " \
+    && apt-get update && apt-get install -y --no-install-recommends $BUILD_DEPS \
+    && wget -q -O /tmp/get-pip.py https://bootstrap.pypa.io/pip/2.7/get-pip.py \
+    && python2.7 /tmp/get-pip.py \
+    && rm /tmp/get-pip.py \
+    && pip install virtualenv \
```

Comment:
Can we do `python2.7 -m pip install ...` instead. And similarly below `/venv/bin/python -m pip install ...`

Reply:
I don't believe so, unfortunately...

```
$ podman run -it ubuntu:20.04
root@36152805e9bb:/# apt update
<snip>
root@36152805e9bb:/# apt install -y python2.7
<snip>
root@36152805e9bb:/# python2.7 -m pip
/usr/bin/python2.7: No module named pip
```

Reply:
Ah! It was just this line you're wondering about isn't it, not how it's installed. 😊 Yes we can definitely do that.